dotCMS/src/main/java/com/dotmarketing/portlets/folders/business/FolderAPIImpl.java

```diff
@@ -92,7 +92,7 @@
 				CollectionsUtils.set(Config.getStringArrayProperty("RESERVEDFOLDERNAMES",
 					new String[]{"WEB-INF", "META-INF", "assets", "dotcms", "html", "portal",
 							"email_backups",
-							"DOTLESS", "DOTSASS"})
+							"DOTLESS", "DOTSASS", "dotAdmin", "custom_elements"})
```

Comment:
test?

Reply:
tests for this logic have already been implemented :)